src/FluentHub/Views/Repositories/Insights/InsightsPage.xaml.cs

```diff
@@ -29,10 +29,6 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
             var currentItem = navigationService.TabView.SelectedItem.NavigationHistory.CurrentItem;
             currentItem.Header = "Insights";
             currentItem.Description = "Insights";
-
-            currentItem.Url = url;
-            currentItem.DisplayUrl = $"{pathSegments[0]} / {pathSegments[1]} / Insights";
-
             currentItem.Url = $"{url}";
             currentItem.DisplayUrl = $"{pathSegments[3]} / {pathSegments[4]} / Insights";
 
```

Comment:
Dont change.....

Reply:
Okay - wait you mean turn it back to [0] [1]?

Reply:
yep.

Reply:
pathSegments starts with like FluentHub/FluentHub/pulse

Reply:
You want me to change it from / Insights to / Pulse?
Personally Insights is easier to understand than Pulse